lib/components/header.js

```diff
@@ -109,30 +108,42 @@ export default class Header extends PureComponent {
 
     return (
       <header
-        className={css('header', isMac && 'headerRounded')}
+        className={`header_header ${isMac && 'header_headerRounded'}`}
         onMouseDown={this.handleHeaderMouseDown}
         onDoubleClick={this.handleMaximizeClick}
       >
         {!isMac && (
-          <div className={css('windowHeader', props.tabs.length > 1 && 'windowHeaderWithBorder')} style={{borderColor}}>
+          <div
+            className={`header_windowHeader ${props.tabs.length > 1 && 'header_windowHeaderWithBorder'}`}
+            style={{borderColor}}
+          >
             {hambMenu && (
               <svg
-                className={css('shape', (left && 'hamburgerMenuRight') || 'hamburgerMenuLeft')}
+                className={`header_shape ${(left && 'header_hamburgerMenuRight') || 'header_hamburgerMenuLeft'}`}
                 onClick={this.handleHamburgerMenuClick}
               >
                 <use xlinkHref="./renderer/assets/icons.svg#hamburger-menu" />
               </svg>
             )}
-            <span className={css('appTitle')}>{title}</span>
+            <span className="header_appTitle">{title}</span>
             {winCtrls && (
-              <div className={css('windowControls', left && 'windowControlsLeft')}>
-                <svg className={css('shape', left && 'minimizeWindowLeft')} onClick={this.handleMinimizeClick}>
+              <div className={`header_windowControls ${left ? 'header_windowControlsLeft' : ''}`}>
+                <svg
+                  className={`header_shape ${left && 'header_minimizeWindowLeft'}`}
+                  onClick={this.handleMinimizeClick}
+                >
                   <use xlinkHref="./renderer/assets/icons.svg#minimize-window" />
                 </svg>
-                <svg className={css('shape', left && 'maximizeWindowLeft')} onClick={this.handleMaximizeClick}>
+                <svg
+                  className={`header_shape ${left && 'header_maximizeWindowLeft'}`}
+                  onClick={this.handleMaximizeClick}
+                >
                   <use xlinkHref={maxButtonHref} />
                 </svg>
-                <svg className={css('shape', 'closeWindow', left && 'closeWindowLeft')} onClick={this.handleCloseClick}>
+                <svg
+                  className={`header_shape header_closeWindow ${left && 'header_closeWindowLeft'}`}
```

Comment:
won't this add the class name `.false` to this?
![image](https://user-images.githubusercontent.com/5027156/37421971-ab59bff6-27ba-11e8-88e0-3f2a37168490.png)

maybe use https://github.com/JedWatson/classnames or similar 🤔

Reply:
I'm not sure we have enough of this case to use classnames. But could do it in another PR later.
I'll just change them to ternaries for now